### ISSUE TITLE
Squiz.SelfMemberReference.NotUsed not detected if leading namespace separator is used

### ISSUE THREAD

**Issue by nkovacs** [squizlabs/PHP_CodeSniffer#1072]
`\Foo` should be replaced with `self`, but it's not detected by the sniff because of the leading namespace separator:

```php
class Foo
{
    public static function bar()
    {
        \Foo::baz();
    }

    public static function baz()
    {
        echo 'foobar';
    }
}

Foo::bar();
```

Failing test: https://github.com/nkovacs/PHP_CodeSniffer/commit/284739882ec692a465fb8896044e9dff1cc510ba

**Comment by gsherwood** [comment]
Thanks for the report. Fixed now.
